openzwavemqtt/util.py

```diff
@@ -0,0 +1,215 @@
+"""Utility functions and classes for OpenZWave."""
+from typing import Any, Dict, List
+
+from .const import (
+	ATTR_LABEL,
+	ATTR_MAX,
+	ATTR_MIN,
+	ATTR_OPTIONS,
+	ATTR_PARAMETER,
+	ATTR_POSITION,
+	ATTR_TYPE,
+	ATTR_VALUE,
+	CommandClass,
+	ValueGenre,
+	ValueType,
+)
+from .manager import OZWManager
+from .models.node import OZWNode
+
+
+def get_node_from_manager(
+    manager: OZWManager, instance_id: int, node_id: int
+) -> OZWNode:
+    """Get OZWNode from OZWManager."""
+    instance = manager.get_instance(instance_id)
+    if not instance:
+        raise KeyError(f"OZW Instance {instance_id} not found")
```

Comment:
Create a library specific exception for this error in a module `exceptions.py`. Make a library specific base exception first that inherits `Exception`. Then inherit that base exception in this not found error.